src/pages/shop/Shop.tsx

```diff
@@ -1,15 +1,193 @@
-import Banner from '../../components/banner/Banner';
-import React from 'react';
+import Banner from '../../components/Banner/Banner';
+import React, { useEffect, useState } from 'react';
+import {Button,  Flex, Stack, Wrap, WrapItem } from '@chakra-ui/react';
+import { BsFillGridFill, BsFillGrid3X3GapFill } from 'react-icons/bs';
+import { ChevronRight } from 'react-bootstrap-icons';
+import { MdOutlineKeyboardArrowDown } from 'react-icons/md';
+
+import { TfiLayoutGrid4Alt } from 'react-icons/tfi';
+import { VscFilter } from 'react-icons/vsc';
+import { GrFormClose } from 'react-icons/gr';
+import '../shop/shop.scss';
+import { Link } from 'react-router-dom';
+
+
 
 export default function Shop() {
+
+  const [isActive, setActive] = useState(false);
+  const [isFilter, setFilter] = useState(false);
+ 
+
+  
+  const handleToggle = () => {
+    setActive(!isActive);
+  };
+  const handleFilterToggle = () => {
+    setFilter(!isFilter);
+  };
+
+
   return (
-    <section className="Shop">
+    <section className="shop">
       <Banner
         title="Shop"
         pageName="Shop"
         coverPhoto="https://casona.familab.net/wp-content/uploads/2021/09/heading_bg_dark.jpg"
         sliderExist={true}
+        bannerHeight="440px"
       />
+      <div className="container">
+        <div className="shop-control">
+          <Flex gap="20"  justifyContent="space-between">
+            <button className="filtering" onClick={handleFilterToggle}>
+              <div className="filterDropdown">
```

Comment:
for class names we have kebab-case naming convention (like shop-control)
could you please rename your classes?